shared/pubsub.js

```diff
@@ -453,14 +453,7 @@ const defaultMessageHandlers = {
       }
       case REQUEST_TYPE.PUSH_ACTION: {
         const { actionType, message } = data
-        const errorHandler = actionType ? pushClientErrorHandler[actionType] : null
-
         console.warn(`[pubsub] Received ERROR for PUSH_ACTION request with the action type '${actionType}' and the following message: ${message}`)
```

Comment:
@taoeffect I was about to add `console.warn` below as we discussed, but then realised there is already one up here. So decided to just remove the duplicate part. (hope it sounds good!)